distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -84,4 +84,4 @@ GC_OPTS="-Xlog:gc*=info:file=${DDF_HOME}/data/log/gc.log:time,uptime,level,tags:
 
 export KARAF_OPTS="-Dfile.encoding=UTF8"
 
-export JAVA_OPTS="-Xms2g -Xmx6g -XX:+UnlockDiagnosticVMOptions -Dddf.home=${DDF_HOME} -Dddf.home.perm=${DDF_HOME}/ -XX:+DisableAttachMechanism $JAVA_ERROR_OPTS $GC_OPTS"
+export JAVA_OPTS="-Xms2g -Xmx6g -XX:+UnlockDiagnosticVMOptions -Dddf.home=${DDF_HOME} -Dddf.home.perm=${DDF_HOME}/ -XX:+DisableAttachMechanism $JAVA_ERROR_OPTS $GC_OPTS --add-opens java.base/java.lang.reflect=ALL-UNNAMED --add-opens java.base/java.text=ALL-UNNAMED --add-opens java.desktop/java.awt.font=ALL-UNNAMED"
```

Comment:
❓ what are the additional JVM args needed for?

Reply:
It disables runtime access warnings for those libraries. I am curious as to why we need it for `java.awt.font`.

Reply:
This is what I said about it in the commit message.

> - Added JAVA_OPTs to suppress XStream reflection module warnings which
>   were printing to standard out during install.

This is to fix an issue where XStream was printing out to standard out with this warning.

> WARNING: Illegal reflective access by com.thoughtworks.xstream.core.util.Fields (file:/private/tmp/ddf/ddf-2.28.0-SNAPSHOT/data/cache/org.eclipse.osgi/586/0/bundleFile) to field java.lang.reflect.Proxy.h
> WARNING: Please consider reporting this to the maintainers of com.thoughtworks.xstream.core.util.Fields
> WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
> WARNING: All illegal access operations will be denied in a future release